tests/gpu/lambda.cpp

```diff
@@ -46,6 +61,12 @@ int main(int argc, char **argv) {
 
   std::cout << "x[0] = " << x[0] << "\n";
 
+  register_run(
+      2, [ =, c = proteus::jit_variable(a) ] __device__
+      __attribute__((annotate("jit"))) (int i) { x[i] = c; });
+
+  std::cout << "x[0:1] = " << x[0] << "," << x[1] << "\n";
```

Comment:
OK, I think it's this printout that's failing. We should change the `CHECK:` line

Reply:
ah! yes, I did try locally, I will fix